src/modules/birthday.py

```diff
@@ -10,4 +11,5 @@ def __init__(self, value):
         try:
             self.birthday = datetime.strptime(value, "%d.%m.%Y")
         except ValueError:
-            raise ValueError("Invalid date format. Use DD.MM.YYYY")
+            error_text = "Invalid date format. Use DD.MM.YYYY"
+            raise ValueError(update_text_color(error_text, EnumColoramaText(3)))
```

Comment:
Maybe we will use enums like this EnumColoramaText.WARNING? It weems more human like and it will be easier to read. IMHO